src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -235,6 +236,8 @@ public void writeThrough(Post post){
   public void writeThrough(Comment comment){
```

Comment:
writeThrough post, comment are the two new functions with respect to really original file